bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WWAToLNuLNuA_4f_NLO/HA_WWA_YUKAWA/CCbar_HGamma/CCbar_HGamma.patch

```diff
@@ -0,0 +1,93 @@
+--- a/Template/NLO/Source/extra_weights.f	2019-10-16 13:39:36.000000000 +0200
++++ b/Template/NLO/Source/extra_weights.f	2021-01-01 00:00:00.000000000 +0200
+@@ -20,9 +20,9 @@
+          character(len=1024),allocatable :: n_ctr_str(:)
+         
+ c input of cpower (checked against calculated value)
+-         double precision,parameter :: cpowerinput=0d0
++         double precision,parameter :: cpowerinput=2d0
+ c switch for running muR-dependent factor runfac=1(running)/0(fixed)
```

Comment:
can you comment on the purpose of the patch and how it was tested/validated against 2.6.5?

Reply:
@agrohsje @sihyunjeon Sorry for the late reply, we got this patch from the Higgs+charm group[1] and we use this patch to achieve dynamic scale choice (to run the yc parameter inside MG5) when we generate mc samples because 
the yc value choice will influence cross-section a lot. About u/d/s process, because we use a special model (because loop_sm can not generate u/d/s processes), we have not found a patch to run those yukawa values in this model. but recently we are studying the yukawa values influences on u/d/s processes by PDG public up/down mass values, and probably we will consider them as one of the uncertainties. About the charm process, we plan to use the patch directly.

[1][https://zguan.web.cern.ch/zguan/WWA_gridpack/hgamma/210928.pdf](https://zguan.web.cern.ch/zguan/WWA_gridpack/hgamma/210928.pdf)